backend/packages/db/index.js

```diff
@@ -0,0 +1,326 @@
+// @flow
```

Comment:
This code should also be open sourced asap, as other arangodb devs might find it useful.